### PR TITLE
Sdp mediadrm support

### PR DESCRIPTION
**Pull request by petegriffin**
Adds support to the mediadrm plugin for decrypting into a SDP allocated buffer. Relies on a corresponding pull request to the ClearKey TA

### CODE REVIEW

**Review by andrey-konovalov — APPROVED** [review]
Looks good to me

File: AesCtrDecryptor.cpp [hunk 1]
```diff
@@ -61,8 +62,13 @@ android::status_t AesCtrDecryptor::decrypt(const android::Vector<uint8_t>& key,
         const SubSample& subSample = subSamples[i];
 
         if (subSample.mNumBytesOfClearData > 0) {
+	  if (secure)
+	    TEE_copy_secure_memory(destination, source,
+				   subSample.mNumBytesOfClearData, offset);
```
Comment (ajutras-linaro) [inline AesCtrDecryptor.cpp]: Is this API defined when USE_AES_TA is not defined?

**Review by ajutras-linaro — CHANGES_REQUESTED** [review]
See inline comment.

File: AesCtrDecryptor.cpp (hunk 1 above)
Comment (petegriffin) [reply AesCtrDecryptor.cpp]: No it isn't good point. Will fix.

**Pull request merged**